src/analyses/invariant_set.h

```diff
@@ -22,6 +22,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "interval_template.h"
 
+#define nullptr_exceptiont(str) str
```

Comment:
Changed the PR title in a desperate attempt to introduce dependency tracking into github...